CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.0.12
+* Fix translation in Vietnamese
+* Add localizations for Hebrew
+
 ## 1.0.12
```

Comment:
Seems like you forgot to remove the old entry ?

Reply:
Not sure how that happened. I added `Add localizations for Hebrew` at 1.0.12 (not into a new version because I see `1.0.12` was not released yet).